package.json

```diff
@@ -164,6 +164,9 @@
           "@types/ws": "*"
         }
       }
+    },
+    "patchedDependencies": {
+      "@chromatic-com/playwright": "patches/@chromatic-com__playwright.patch"
```

Comment:
Shouldn't we set _exact version_ requirement on chromatic dependency if we patch it?

Reply:
Pinned the version to `0.10.2`.